tests/array-analysis.test.js

```diff
@@ -0,0 +1,7 @@
+const arrayAnalysis = require('../src/array-analysis');
+
+test('average of array items', () => {
+  expect(arrayAnalysis([1, 2, 3, 4])).toEqual({
```

Comment:
- [ ] Add another test case for this method. You should have at least 2 test cases, otherwise you can't tell if it's just passing the first by a fluke or hardcode.